landing.css

```diff
@@ -0,0 +1,219 @@
+*,
+::before,
+::after {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    height: 100vh;
+}
+
+a {
+    text-decoration: none;
+    color: var(--text-color);
+}
+
+html {
+    font-size: 62.5%;
+}
+
+.underline {
+    height: 0.5rem;
+    width: 30%;
+    background-color: var(--primary-color);
+    margin: auto;
+}
+
+/* grid */
+.landing-container {
+    min-height: 100%;
+    display: grid;
+    grid-template-columns: 200px repeat(4, 1fr) 200px;
+    grid-template-rows: repeat(5, auto);
+    grid-template-areas:
+        "nav nav nav nav nav nav"
+        "contentarea contentarea contentarea contentarea contentarea contentarea"
+        "contentarea contentarea contentarea contentarea contentarea contentarea"
+        "contentarea contentarea contentarea contentarea contentarea contentarea"
+        "footerarea footerarea footerarea footerarea footerarea footerarea";
+}
+
+nav {
+    grid-area: nav;
+}
+
+footer {
+    grid-area: footerarea;
+    margin-top: auto;
+}
+
+/* title section */
+.title-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-top: auto;
+    margin-bottom: auto;
+    margin-top: 5rem;
+    line-height: 5rem;
+}
+
+.title-container .name {
+    margin-left: 0;
+    font-size: 5rem;
+}
+
+.title-container .btn {
+    margin: 1rem 0;
+    font-weight: 500;
+    border: none;
+}
+
+.title-container .btn:hover {
+    background-color: var(--primary-color-dark);
+    transition: 0.2s ease-in-out;
+}
+
+.title-container p {
+    font-size: 2rem;
+    font-weight: 400;
+    color: var(--content-color);
+    text-align: center;
+}
+
+/* brand and products section */
+
+.section-title {
+    width: 100%;
+    text-align: center;
+    margin-top: 5rem;
+    font-size: 3rem;
+    font-weight: 500;
+    padding: 1rem;
+    color: var(--content-color);
+}
+
+.showcase-container {
+    grid-area: contentarea;
+}
+
+.categories-section {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 2rem;
+    width: 100%;
+    flex-wrap: wrap;
+}
+
+.category {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 1rem 1rem;
```

Comment:
You have used flex-center multiple times in your code. You can just create a utility class for this. It will reduce redundancy.

Reply:
sure I will do that